oaknational/Oak-Web-Application
# [CUR-567] Units overview page to docx generation

## Description

- List of changes

## Issue(s)

`CUR-567`

## How to test

1. Go to https://deploy-preview-2434--oak-web-application.netlify.thenational.academy/teachers/curriculum/docx-poc/select
2. Choose a subject via the autoComplete
3. Download the docx template - linked on the page
4. Download the rendered document
5. Check the units overview page is present and correct for each year in keystage

## Screenshots

<img width="529" alt="Screenshot 2024-05-20 at 12 18 37" src="https://github.com/oaknational/Oak-Web-Application/assets/235915/5ba7307b-ed1c-492c-903f-c0f9ccba0847">
